src/components/includes/includes--footer-related.njk

```diff
@@ -0,0 +1,84 @@
+{%- set buttons = [
+  {
+    title: "Contact us",
+    url: "/about/contact.html"
+  },
+  {
+    title: "Give now",
+    url: "https://www.myiu.org/give-now?utm_source=iu-edu&amp;utm_medium=referral&amp;utm_campaign=gateways&amp;utm_content=give-now-footer"
+  }
+] -%}
+{%- set categories = [
+  {
+    title: "Services",
+    links: [
+      {
+        title: "Canvas",
+        link: "https://canvas.iu.edu"
+      },
+      {
+        title: "One.IU",
+        link: "https://one.iu.edu"
+      }
+    ]
+  },
+  {
+    title: "Email",
+    links: [
+      {
+        title: "Outlook Web Access",
+        link: "https://uits.iu.edu/exchange"
+      },
+      {
+        title: "Gmail at IU",
+        link: "https://google.iu.edu"
+      }
+    ]
+  },
+  {
+    title: "Find",
+    links: [
+      {
+        title: "People Directory",
+        link: "https://directory.iu.edu/"
+      },
+      {
+        title: "Jobs at IU",
+        link: "https://jobs.iu.edu/"
+      },
+      {
+        title: "Non-discrimination Notice",
+        link: "/nondiscrimination/index.html"
+      }
+    ]
+  }
+] -%}
+<div aria-labelledby="resources-heading" class="rvt-c-footer-related" role="complementary">
```

Comment:
I think we can maybe do a better job of naming these elements of the footer with more clear/semantic names. For example "related" isn't super clear to me. Related to what?

definitely not something I want to hold this PR up for, but something I want to figure out before we release the next alpha. that's something I'm happy to take on. I'll create an internal task for it.